src/main/webapp/view.js

```diff
@@ -23,6 +23,8 @@ function switchView() {
             alert("You do not have permission to switch views.");
             return;
         }
+
+        location.reload();
```

Comment:
Why do you reload here?

Reply:
It's so that the new view is loaded after it is switched. Otherwise it would just stay in the old view.